metadata-ingestion/src/datahub/ingestion/source/powerbi/config.py

```diff
@@ -130,21 +148,27 @@ class PowerBiAPIConfig(EnvBasedSourceConfigBase):
     )
     # Enable/Disable extracting ownership information of Dashboard
     extract_ownership: bool = pydantic.Field(
-        default=True, description="Whether ownership should be ingested"
+        default=True,
+        description="Whether ownership should be ingested. Admin access should be given to the configured "
```

Comment:
"Admin API access is required if this setting is enabled. Note that enabling this may overwrite owners that you've added inside DataHub's web application.". 

Let's also flip this to disabled by default.

Reply:
done